domenetjenester/behandling-prosessering/src/main/java/no/nav/ung/sak/behandling/prosessering/BehandlingProsesseringTjenesteImpl.java

```diff
@@ -116,9 +120,11 @@ private ProsessTaskGruppe doOppfriskingTaskOgFortsattBehandling(Behandling behan
 
         ProsessTaskGruppe gruppe = new ProsessTaskGruppe();
         if (behandling.erYtelseBehandling()) {
-            ProsessTaskData registerdataOppdatererTask = ProsessTaskData.forProsessTask(OppfriskingAvBehandlingTask.class);
-            registerdataOppdatererTask.setBehandling(behandling.getFagsakId(), behandling.getId(), behandling.getAktørId().getId());
-            gruppe.addNesteSekvensiell(registerdataOppdatererTask);
+            if (!oppfriskKontrollbehandlingEnabled) {
```

Comment:
jeg tror ikke det holder.  FortsettBehandlingTask lenger nede tar også av vent. 
https://github.com/navikt/ung-sak/blob/b0f118fa77343975356c02b70fca72c78893d590/domenetjenester/behandling-prosessering/src/main/java/no/nav/ung/sak/behandling/prosessering/task/FortsettBehandlingTask.java#L64

Reply:
Det er så mange metoder i det interfacet... Det er litt krevende å vite kva forskjellen er mellom dei ulike

Reply:
ok. Det er kanskje enda bedre. Bare fjerne og forenkle venting for alle 👍  
Men du vil ha det ut i Q for å test?